### PR TITLE
Adding training visualization to the CVAE tutorial

### PR DESCRIPTION
**Pull request by ucals**
Hey guys,
Before moving to create another tutorial (next one will be the trajectory prediction paper), yesterday I had an idea to a visualization, which I think is pretty neat: check it out!

I added a video showing how the loss progresses throughout the training, and how the CVAE predictions evolve over time, going from blurry to clarity. I think it's cool, fun to watch :)

Unfortunately it cannot be seen directly in the .ipynb file, you have to run make tutorial to see it. Conversely, you can see in [this page](https://ucals.github.io/paper/2020/09/18/conditional-variational-auto-encoder.html#training), it renders exactly like that. Hope you dig it!

@eb8680 , pls let me know if it is right this time...

Cheers!

### CODE REVIEW

**Comment by eb8680** [comment]
Thanks for fixing the git history.  The video looks great and renders correctly on your linked page, but when I run `make tutorial` on your branch locally, I just see an HTML code block. Can you confirm whether this happens to you?

image

File: tutorial/source/cvae.ipynb [hunk 1]
```diff
@@ -303,6 +303,39 @@
...
+   "source": [
+    "## Training\n",
+    "The training code can be found in the [Github repo](https://github.com/pyro-ppl/pyro/tree/dev/examples/cvae). \n",
+    "Click play in the video below to watch how the CVAE learns throughout approximately 40 epochs.\n",
+    "\n",
+    "<p>\n",
+    "    <video muted controls loop width=\"100%\">\n",
+    "        <source src=\"https://ucals-github.s3-sa-east-1.amazonaws.com/cvae_animation.mp4\" type=\"video/mp4\">\n",
+    "        <source src=\"https://ucals-github.s3-sa-east-1.amazonaws.com/cvae_animation.webm\" type=\"video/webm\">\n",
+    "    </video>\n",
```
Comment (fritzo) [inline tutorial/source/cvae.ipynb:318]: maybe try avoiding the indentation? Maybe nbsphinx misinterprets the spaces as a code block?

**Comment by ucals** [comment]
Hummm... that's odd, it works here:

Screen Shot 2020-09-21 at 13 55 14

I'll remove the indentation as suggested by @fritzo , one sec...

**Comment by ucals** [comment]
Just removed the indentation... in my machine it is rendering as expected:
Screen Shot 2020-09-21 at 14 14 13
:

Pls let me know if it works... cheers!

**Comment by eb8680** [comment]
> Just removed the indentation... in my machine it is rendering as expected:

That fixed it on my end as well.

**Review by eb8680 — APPROVED** [review]
LGTM

**Pull request merged**